base/applications/notepad/lang/sl-SI.rc

```diff
@@ -160,27 +160,27 @@ Vnesite besedilo in poskusite znova."
 Ali želite ustvariti novo datoteko?"
     STRING_NOTSAVED "Vsebina datoteke '%s'je bila spremenjena.\n\n\
 Ali želite shraniti spremembe?"
-    STRING_NOTFOUND "Ni mogoèe najti '%s'."
-    STRING_OUT_OF_MEMORY "Na voljo ni dovolj pomnilnika, da bi bilo mogoèe dokonèati to \
-operacijo.\nÈe ga želite sprostiti, konèajte enega ali veè programov in poskusite znova."
-    STRING_CANNOTFIND "Cannot find '%s'"
+    STRING_NOTFOUND "Datoteke '%s' ni mogoče najti."
+    STRING_OUT_OF_MEMORY "Na voljo ni dovolj pomnilnika, da bi bilo mogoče dokončati to \
+operacijo.\nČe ga želite sprostiti, končajte enega ali več programov in poskusite znova."
+    STRING_CANNOTFIND "Ni rezultatov za %s"
```

Comment:
Missing quotes around %s — it's useful if you're searching for whitespace or unprintable characters (if one can do that in notepad).